bumble/smp.py

```diff
@@ -764,7 +764,11 @@ def __init__(
         self.peer_io_capability = SMP_NO_INPUT_NO_OUTPUT_IO_CAPABILITY
 
         # OOB
-        self.oob_data_flag = 0 if pairing_config.oob is None else 1
+        self.oob_data_flag = 0
```

Comment:
suggestion: `self.oob_data_flag = 1 if pairing.oob and pairing_config.oob.peer_data else 0`